editoast/src/main.rs

```
@@ -116,11 +116,12 @@ fn init_tracing(mode: EditoastMode, telemetry_config: &client::TelemetryConfig)
             Some(layer)
         }
     };
-    tracing_subscriber::registry()
+
+    let _ = tracing_subscriber::registry()
         .with(telemetry_layer)
         .with(env_filter_layer)
         .with(fmt_layer)
-        .init();
+        .try_init();
```

We might want to ignore an error for tests, but I believe we want to crash the application at startup in case of an error in production. It would not be good to start our application in production with an incorrect subscriber. Maybe `init_tracing` can return an error?

---

https://github.com/OpenRailAssociation/osrd/commit/c361ea8704094ed3e9967804a436922df5434e76